### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by kr-sarvesh**
https://redstorez.netlify.app/screens/wishlist/wishlist.html

attached live preview

love to see reviews

### CODE REVIEW

**Review by Balakumar3018 — APPROVED** [review]
Everything looks good.
Suggestion - add remove from list option in wishlist

File: screens/authentication/auth.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  cursor: pointer;
+  width: 100px;
+  display: inline-block;
+}
+.form-btn {
+  display: inline-block;
+}
+.form-container form {
+  max-width: 300px;
+  padding: 0 20px;
```
Comment (DevNirah7) [inline screens/authentication/auth.css:32]: Suggestion: Can use relative values for padding and width.

**Review by DevNirah7 — APPROVED** [review]
1. Can use relative values for height, widths etc.

Rest of the code looks clean. Good to commit!

File: screens/authentication/auth.css (hunk 1 above)
Comment (kr-sarvesh) [reply screens/authentication/auth.css:32]: ok

**Pull request merged**